test/cmd.js

```diff
@@ -115,7 +115,7 @@ describe('express(1)', function () {
       it('should create basic app', function (done) {
         run(ctx.dir, [], function (err, output) {
           if (err) return done(err)
-          assert.equal(parseCreatedFiles(output, ctx.dir).length, 17)
```

Comment:
Great work and iniciative @asimpletune!! I want to contribute!
Btw, for `es6/es2015` - `babel` we need to transpile the code to `es5` (nodejs compatible) because `babel-node` and `babel-core/register` aren't for production usage (see [https://github.com/babel/example-node-server/issues/13](see)). More info [here](https://github.com/babel/example-node-server#getting-ready-for-production-use), and [here](https://medium.com/@Cuadraman/how-to-use-babel-for-production-5b95e7323c2f#.bwjjho9m2).